modules/viewing-performance-metrics-for-deployed-model.adoc

```diff
@@ -0,0 +1,72 @@
+:_module-type: PROCEDURE
+
+[id="viewing-performance-metrics-for-model-server_{context}"]
+= Viewing performance metrics for a deployed model
+
+[role='_abstract']
+
+You can monitor the following metrics for a specific model that is deployed on the single-model serving platform:
+
+* *Number of requests* - The number of HTTP requests that have failed or succeeded for a specific model.
+* *Average response time (ms)* - The average time it takes a specific model to respond to requests.
+* *CPU utilization (%)* - The percentage of the CPU's capacity that is currently being used by a specific model.
+* *Memory utilization (%)* - The percentage of the system's memory that is currently being used by a specific model.
```

Comment:
```suggestion
* *Memory utilization (%)* - The percentage of the  model deployment's memory limit that is currently being used by a specific model.
```

Reply:
@VedantMahabaleshwarkar What does _model deployment_ mean here?

Reply:
Pod if there's only 1 replica, pods if there are multiple replicas. But what I specifically mean is the Openshift `Deployment` that represents the model. The deployment will have resource `requests` and `limits`. The % we show is the % relative to the `limit` that is set in the deployment

Reply:
@VedantMahabaleshwarkar I think that model deployment might sound confusing - proposing an alternative, let me know your thoughts: 


* *CPU utilization (%)*: The percentage of the CPU limit per model replica that is currently utilized by a specific model.
* *Memory utilization (%)* - The percentage of memory limit per model replica that is currently utilized by a specific model.

Based my assumption on this setting: 

<img width="810" alt="Screenshot 2024-08-08 at 9 53 06 AM" src="https://github.com/user-attachments/assets/1f1ffe3e-69c4-4e60-ba17-3fe8b7febcd4">